skyportal/tests/api/test_sources.py

```diff
@@ -771,10 +772,9 @@ def test_object_last_detected(
     )
     assert status == 200
     assert data["status"] == "success"
-    assert (
-        data["data"]["last_detected_at"]
-        == arrow.get((90000.0 - 40_587) * 86400.0).isoformat()
-    )
+    assert arrow.get(
+        Time(data["data"]["photstats"][-1]["last_detected_mjd"], format="mjd").datetime
+    ) == arrow.get((90000.0 - 40_587) * 86400.0)
```

Comment:
I think you may be able to just compare MJD to MJD.

Reply:
I have a small preference since datetime is what we use in the app.

Reply:
But aren't you just converting both sides of this inequality to MJD? Do you feel this helps make sure the datetime is correct?

Reply:
Actually I checked and it is calculating unix time on the right hand side and converting, so this way is probably best.